### ISSUE TITLE
Fast double click to play with MPV engine, even the existing sound, but the video has been in a certain frame

### ISSUE THREAD

**Issue by gladiatorplus** [mpv-player/mpv#8055]
- mpv 0.32
- arm
- Window Manager and version:wayland
- GPU driver and version
- Possible screenshot or video of visual glitches

If you're not using git master or the latest release, update.
Releases are listed here: https://github.com/mpv-player/mpv/releases

### Actual behavior
Quickly open different files and observe the phenomenon
### Log file

Make a log file made with -v -v or --log-file=output.txt, paste it to
https://0x0.st/ or attach it to the github issue, and replace this text with a
link to it.

The issue will be closed for ignoring the issue template.
[mpv.log](https://github.com/mpv-player/mpv/files/5167953/mpv.log)

**Comment by wm4** [comment]
```
[  11.668][v][vo/libmpv] mpv_render_context_render() not being called or stuck.
```
No idea what your problem is, but it seems you're using libmpv, but not calling mpv_render_context_render as required.

**Comment by gladiatorplus** [comment]
Yes,you're right. what are the possible causes of this problem? please tell me,I thought about it for a long time and I didn't solve it

**Comment by wm4** [comment]
You probably need to set this:
https://github.com/mpv-player/mpv/blob/master/libmpv/render.h#L614

**Comment by gladiatorplus** [comment]
Thank you.

**Comment by gladiatorplus** [comment]
It can be useful. I'll try
